server/image_pull_test.go

```diff
@@ -66,137 +46,14 @@ var _ = t.Describe("ImagePull", func() {
 			Expect(response).NotTo(BeNil())
 		})
 
-		It("should succeed when already pulled", func() {
```

Comment:
Outright removing tests makes me worried. Why is that OK? It might well _be OK_, maybe the test is directly redundant, but the PR doesn’t say.

(Based on past experience, I am very skeptical of the mock tests, they tend to trigger very unrealistic code paths without anyone noticing, but still…)

Reply:
It’s fine because the tested code paths do not exist any more due to the simplification.